draft-ietf-taps-interface.md

```diff
@@ -1812,30 +1829,29 @@ Type:
 Default:
 : Disabled
 
-This property specifies how long to wait before deciding that an active Connection has
+This property specifies how many seconds to wait before deciding that an active Connection has
```

Comment:
Here and below I propose that these times are in seconds. Is that right, or do we want the unit of duration to be implementation-specific?

Reply:
Let's instead say that the representation of duration is implementation specific?

Reply:
Say something about the representation of duration being implementation-specific.